quickstart/docker/image/ziti-cli-functions.sh

```diff
@@ -279,6 +279,36 @@ function checkPrereqs {
   fi
 }
 
+function _portCheck {
+  if [[ "${1-}" == "" ]] || [[ "${2-}" == "" ]]; then
```

Comment:
i feel like it'd be helpful to inform the users that the configured ports are being checked for open-ness. it's also only checking that they are available LOCALLY, not 'remotely' (yet) but checking that the 'actual' firewall is open to the 'advertised' host would be a logical next addition here imo